versioned_docs/version-latest/learn/monitoring.md

```diff
@@ -0,0 +1,723 @@
+      - targets: ['localhost:9615']
+  - job_name: "FarmerServer"
+    static_configs:
+      - targets: ["localhost:9100"]
+
+```
+
+Once modifications are done in the `prometheus.yml` file, remember to reload the service:
+
+```bash
+sudo systemctl daemon-reload
+```
+
+Ensure to provide names for the `job_name`.
+
+Please remember to delete archives and all extracted files from the home directory after distributing all components:
+
+```bash
+cd ~
+rm -r prometheus*
+```
+
+---
+
+Here's a revised version:
+
+---
+
+### Creating a Service for Prometheus
+
+Open the service file using your preferred text editor:
+
+```bash
+sudo vi /etc/systemd/system/prometheus.service
+```
+
+Add the following configuration to the file:
+
+```bash
+[Unit]
+Description=Prometheus
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+User=prometheus
+Group=prometheus
+Type=simple
+ExecStart=/usr/local/bin/prometheus \
+    --config.file /etc/prometheus/prometheus.yml \
+    --storage.tsdb.path /var/lib/prometheus/ \
+    --web.console.templates=/etc/prometheus/consoles \
+    --web.console.libraries=/etc/prometheus/console_libraries
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Remember to reload the daemon after making any service changes:
+
+```bash
+sudo systemctl daemon-reload			
+# Run this whenever any service configuration changes
+```
+
+---
+Here's a revised version for better readability:
+
+---
+
+### UFW Configuration
+
+On the Prometheus Server:
+```bash
+sudo ufw allow from <farmer-server-1-ip> to any proto tcp port 9100 comment 'Exporter-farmer-1'
+sudo ufw allow from <farmer-server-2-ip> to any proto tcp port 9100 comment 'Exporter-farmer-2' 
+```
+Repeat this configuration for other farmer servers as necessary.
+
+On the Monitored Servers:
+```bash
+sudo ufw allow from <prometheus-server-ip> to any proto tcp port 9100 comment 'Exporter-PrometheusServer'
+```
+
+### Service Management Commands:
+
+```bash
+sudo systemctl start prometheus			
+# Start Prometheus service
+
+sudo systemctl status prometheus			
+# Check Prometheus service status
+
+sudo systemctl enable prometheus			
+# Enable Prometheus service for autostart
+
+sudo systemctl disable prometheus			
+# Disable autostart for Prometheus service
+```
+
+---
+
+Here's a revised version for improved readability:
+
+---
+
+## 3. Grafana Server
+
+You have the option to utilize Grafana Cloud instead of managing and scaling your instance of Grafana. [Sign up for a free account](https://grafana.com/auth/sign-up?pg=download&plcmt=box-right), offering free access to 10k metrics, 50GB logs, 50GB traces, and more.
+
+### Installation Guide - Latest Open Source Grafana Release (APT repository)
+
+:::note
+Choose between:
+- Enterprise: Identical to the open-source version but offers additional features unlockable with a license.
+- Open Source: Functionally identical to the Enterprise version without enterprise-specific features.
+:::
+
+### Steps to Download
+
+```bash
```

Comment:
Might be nice to explain what this is doing - that we are trusting Grafana's repository. Possibly link to the official docs this is copied from too.